store/migrations/migration1554131520/migrate.go

```diff
@@ -0,0 +1,14 @@
+package migration1554131520
+
+import (
+	"github.com/jinzhu/gorm"
+	"github.com/smartcontractkit/chainlink/store/models"
+)
+
+// Migration is the singleton type for this migration
+type Migration struct{}
+
+// Migrate adds the run request table
+func (m Migration) Migrate(tx *gorm.DB) error {
+	return tx.AutoMigrate(&models.RunRequest{}).Error
```

Comment:
Seems like the only columns on the table are new ones, so we don't migrate any data or drop it any columns. Is that correct? Also we'd talked about adding a RunRequest record for each JobRun, shouldn't that be included in this migration?

Reply:
Not sure I understand the question, but the only migration we do is to create a new table. We do not modify any existing tables.

We create a `RunRequest` record for every **new** `JobRun`, but you're right we ignore past records. For the purposes of this story and linkstats, we don't need to backfill them because  StatsPusher only pushes future runs.

Do you want me to add it anyways?

Reply:
Backfilling `RunRequest` is coming up.